lib/internal/worker.js

```diff
@@ -24,7 +24,8 @@ util.inherits(MessagePort, EventEmitter);
 const {
   Worker: WorkerImpl,
   getEnvMessagePort,
-  threadId
+  threadId,
+  oninit: oninit_symbol
```

Comment:
I understand why you did it, but do we already use snake case in JS elsewhere?

Reply:
@targos Only in this file ;) I’ve added switching to camelCase for both symbols in fb840e7b64c5f44f70fa0f8bf485c9ca49618ca5 (and used the internal binding I forgot existed), let me know if that sounds good to you

Reply:
perfect. thank you!